python/arcticdb/version_store/library.py

```diff
@@ -576,6 +576,32 @@ def enterprise_options(self) -> EnterpriseLibraryOptions:
             background_deletion=write_options.delayed_deletes
         )
 
+    def stage(
+        self,
+        symbol: str,
+        data: NormalizableType,
+        validate_index=True,
+        sort_columns: List[str] = None
+    ):
+        """
+        Write a staged data chunk to storage, that will not be visible until finalize_staged_data is called on
+        the symbol. Equivalent to write() with staged=True.
+
+        Parameters
+        ----------
+        symbol : str
+            Symbol name. Limited to 255 characters. The following characters are not supported in symbols:
+            ``"*", "&", "<", ">"``
+        data : NormalizableType
+            Data to be written. To write non-normalizable data, use `write_pickle`.
+        validate_index:
+            Check that the index is sorted prior to writing. In the case of unsorted data, throw an UnsortedDataException
```

Comment:
Do we want to include this argument? I guess if you have `validate_index=True` and no sort columns it's saying you know it's sorted so we don't need to do anything?

Reply:
Yeah, I think you can either tell us you expect it to be sorted (in which case unsorted data is an error), or ask us to sort it for you. If you do neither and then try to finalize without sorting then it's not going to work.